decklinksdk.rb

```diff
@@ -12,13 +12,9 @@ def install
     if build.with?("macOS") && build.with?("linux")
       puts "Please select only one build option."
       puts "To manually install a specific version, use the flags --with-macOS or --with-linux"
-    elsif build.with?("linux")
+    elsif build.with?("linux") || RUBY_PLATFORM.include?("linux")
```

Comment:
Wouldn't this change make the logic fail for installing the macOS build on linux as it would always evaluate to true at this step? (I know the ability to manually select the build for a different system isn't super important, but it would seem that if this change were made that the option might as well be eliminated entirely)

Reply:
Hmm… I will run more tests. Thanks a lot!

Reply:
Yet, what would be a reason for installing the macOS build on Linux?

Reply:
Not many reasons for sure! It would certainly be a boutique use case, but I have done things like compile Windows targeted FFmpeg builds on linux to help people who were having problems compiling it themselves, so I always like to have options just in case (but that is admittedly personal preference).

That is why I wrote it as it currently is, where it will look for user choice and then default to the system that is running if there were no user options. It would definitely simplify the code to remove that choice, but in that case the option flags `--with-linux` and `--with-macos` should probably be taken out as well, as if the install was automatically tied to the running OS they would no longer serve a purpose.

Reply:
I understand your point, then let us keep it as it is. Sorry for the noise.